### PR TITLE
ras_a: POI_REPORT: remove unused/irrelevant fields to conserve bandwidth

### PR DESCRIPTION
**Pull request by TSC21**
Removes 113 bytes from the message (fields that are either unused on the current applications or are irrelevant to propagate through MAVLink), in order to save some bandwidth when multiple POI's are being reported. Sets also a max of 5Hz per POI reporting.

### CODE REVIEW

**Comment by TSC21** [comment]
@thomas-watters-skydio @radarku @dskyle-shieldai for your review and assessment. Please let me know if I am removing a field that is relevant and that should be kept.

File: message_definitions/v1.0/ras_a.xml [hunk 1]
```diff
@@ -397,15 +396,6 @@
...
-      <field type="float[3]" name="approach_velocity" units="m/s" invalid="[NaN]">Recommended NED velocity for vehicle approach to the POI. This can either be determined by the end system where the POI was detected ir by a system forwarding the information to another vehicle. Unknown is NaN, NaN, NaN.</field>
-      <field type="char[32]" name="name">Name of the POI, if the system provides one. NULL terminated string.</field>
-      <field type="char[31]" name="app6_symbol">APP-6(D) standard symbol 30-digit Symbol Identification Code (SIDC) that provides the necessary information to display a tactical symbol. The SIDC is formed with eleven elements which are presented in two sets of ten digits and an additional set of ten digits composed of three elements, which are optional. Any unspecified element should be set to '0'. The way these codes are built can be checked on the Annex A to the APP-6 - NATO Joint Military Symbology, version D. NULL terminated string.</field>
```
Comment (thomas-watters-skydio) [inline message_definitions/v1.0/ras_a.xml:408]: These two are probably relevant?

Comment (TSC21) [reply message_definitions/v1.0/ras_a.xml:408]: I believe I had this conversation before with you guys, and my understanding is that this is not something that should come out of the vehicle, but rather something that needs user input when it arrives to GCS. We already use the classification field to in someway do that sort of assessment. Military symbology can definitely be the operator to add and classify IMHO.
@SamGrant-Auterion @zach-grant @radarku what's your view of this?

Comment (thomas-watters-skydio) [reply message_definitions/v1.0/ras_a.xml:408]: more so we can identify if its a person/car (both LAND), but can't identify friendly/foe, so for all of our POIs we can report UNKNOWN LAND

**Review by thomas-watters-skydio — APPROVED** [review]

File: message_definitions/v1.0/ras_a.xml [hunk 2]
```diff
@@ -381,7 +381,6 @@
       <field type="uint8_t" name="confidence_classification" units="%" invalid="UINT8_MAX">Confidence level of the POI classification. 0..100, UINT8_MAX when unknown.</field>
       <field type="uint8_t" name="confidence_localization" units="%" invalid="UINT8_MAX">Confidence level of the POI localization. 0..100, UINT8_MAX when unknown.</field>
       <field type="uint16_t" name="ttl" units="s">Time to live: If this time has elapsed since last update, the POI should be deleted on the receiver side. A value of 0 should indicate no timeout.</field>
-      <field type="uint8_t" name="status_flags">Bitmask for POI status. Bit 1: POI is in focus on camera, Bit 8: POI has been cleared and should be deleted.</field>
```
Comment (radarku) [inline message_definitions/v1.0/ras_a.xml]: @TSC21 @thomas-watters-skydio We're currently using `status_flags` as a way to indicate _immediate_ (i.e., as opposed to _eventual_ e.g., TTL) removal/deletion of a POI.  Unless we have an alternative approach for immediate deletion, we need to keep this.

File: message_definitions/v1.0/ras_a.xml (hunk 1 above)
Comment (radarku) [reply message_definitions/v1.0/ras_a.xml:408]: We're fine with removing any/all of these.

**Review by radarku — CHANGES_REQUESTED** [review]

File: message_definitions/v1.0/ras_a.xml (hunk 2 above)
Comment (TSC21) [reply message_definitions/v1.0/ras_a.xml]: Alright lets keep this one then.

**Pull request merged**